doc/release/1.17.0-notes.rst

```diff
@@ -36,7 +36,20 @@ Casting from a different floating point precision to float16 used incorrect
 rounding in some edge cases. This means in rare cases, subnormal results will
 now be rounded up instead of down, changing the last bit (ULP) of the result.
 
+Signed zero when using divmod
+-----------------------------
 
+Starting in version 1.12.0, numpy incorrectly return a negatively signed zero
```

Comment:
return -> returned. Maybe "``` ``divmod`` ``` and ``` ``floor_devide`` ```" (add the double ticks also, I think), since while this is clearer from implementation wise, `//` is much more often used (which is why you used it as an example after all).

Reply:
Ok, this has been updated in my latest push.